src/cmd/tools.go

```diff
@@ -86,10 +89,33 @@ var readCredsCmd = &cobra.Command{
 		}
 
 		message.Note(lang.CmdToolsGetGitPasswdInfo)
+		message.Warn(lang.CmdToolGetGitDeprecation)
 		fmt.Println(state.GitServer.PushPassword)
```

Comment:
See other comment (recommend moving to credentials.go)

Reply:
updated with [484cc98](https://github.com/defenseunicorns/zarf/pull/1220/commits/484cc980a35e6c9f496ff5239b3ac05a3054f98f)